R/aet01.R

```diff
@@ -133,7 +133,11 @@ aet01_lyt <- function(arm_var,
       lbl_vars = lbl_vars
     )
 
-  return(list(ae1 = lyt_ae1, ae2 = lyt_ae2, adsl = lyt_adsl))
```

Comment:
was this change by the styler?

Reply:
I can meet you halfway with:
```r
ret <- list(ae1 = lyt_ae1, ae2 = lyt_ae2, adsl = lyt_adsl)
ret
```
the styler doesn't like `return()`